addon/components/frost-chart-axis-tick-svg.js

```diff
@@ -0,0 +1,84 @@
+/**
+ * Component definition for the frost-chart-axis-tick-svg component
+ */
+
+import Ember from 'ember'
+const {String: EmberString, isNone, run} = Ember
+import computed, {readOnly} from 'ember-computed-decorators'
+import {Component} from 'ember-frost-core'
+import {PropTypes} from 'ember-prop-types'
+
+export default Component.extend({
+
+  // == Dependencies ==========================================================
+
+  // == Keyword Properties ====================================================
+
+  attributeBindings: ['transform'],
+  tagName: 'g',
+
+  // == PropTypes =============================================================
+
+  propTypes: {
+    // options
+    axis: PropTypes.string.isRequired,
+    coordinate: PropTypes.number
```

Comment:
I didn't do this in the previous axis implementation, but this should be required - at least I can't think of a reason that it wouldn't be